### ISSUE TITLE
Add links and disclaimers on site

### ISSUE THREAD

**Issue by OscarLarm** [annk15/MTGKeywordHelper#58]
Similar to the README.md issue, we'll need to add links for credit and disclaimers on the site asap, before making the build. 

Preferrably should be fixed before we merge the latest dev into main.

**Comment by annk15** [comment]
I agree, it shouldn't take to long so we should try that as soon as possible
